### ISSUE TITLE
gather_data.R and parsing_functions.R

### ISSUE THREAD

**Issue by keegan-abdoo** [nstrayer/cv#16]
I'm trying to use this template to revamp my resume.  I've followed along with the tutorial and everything, but don't understand how to use the two sourced R scripts (gather_data.R and parsing_functions.R) which are missing from this repo.

**Comment by SeokLeeUS** [comment]
yes, those two are missing. the author seems don't maintain this repo.
